Model/Order/Invoice/Total/Surcharge.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @by SwiftOtter, Inc. 4/11/17
+ * @website https://swiftotter.com
+ **/
+
+namespace SwiftOtter\ShippingSurcharge\Model\Order\Invoice\Total;
+
+use SwiftOtter\ShippingSurcharge\Model\Surcharge as SurchargeModel;
+
+class Surcharge extends \Magento\Sales\Model\Order\Invoice\Total\AbstractTotal
+{
+    public function collect(\Magento\Sales\Model\Order\Invoice $invoice)
+    {
+        $surcharge = 0;
+        $baseSurcharge = 0;
+
+        /** @var \Magento\Sales\Model\Order\Invoice\Item $item */
+        foreach ($invoice->getAllItems() as $item) {
+            $orderItem = $item->getOrderItem();
+            $orderItemQty = $orderItem->getQtyOrdered();
+
+            if (!$orderItemQty || $orderItem->isDummy() || $item->getQty() < 1) {
```

Comment:
I somewhat agree. In this case, they are coming in as strings, but I don't really care what format. If it's falsy, then I want to skip. In this case, I'm changing the logic to `< 1` as PHP casts the string correctly.